bin/forever

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env zsh
+# This script runs a command forever. Its first argument ($refresh_every)
+# determines if the command should refresh automaticaly.
+#
+# If $refresh_every is greater than 0, the command will refresh
+# automaticaly every X seconds.
+# Else, it refreshes when the user type any letter. If
+# this letter is 'y', it exits.
 
-# This script runs a command and prompts the user.  If the user types
-# 'y' then exit.  If the user types anything else, loop.
+refresh_every="$1"
+shift
 
-local unused
-while clear; date; "$@" ; ! read -q unused; do done
+if [ "$refresh_every" -eq "0" ]; then
```

Comment:
Did you test the refresh_every=0 case?  On my system, with the config file from README.md, $1 is `0.0` so this condition evaluates to false, then line 19 is executed as a busy-wait loop.

```sh
% sh -c '[ 0.0 -eq 0 ]'
zsh:[:1: integer expression expected: 0.0
zsh: exit 2     zsh -c '[ 0.0 -eq 0 ]'
```